validator/src/main.rs

```diff
@@ -1422,6 +1422,11 @@ pub fn main() {
                 .value_name("INDEX")
                 .help("Enable an accounts index, indexed by the selected account field"),
         )
+        .arg(
+            Arg::with_name("accounts_db_caching_enabled")
+                .long("accounts-db-caching-enabled")
+                .help("Enable accounts caching"),
+        )
```

Comment:
nits: (definitely could be done as a follow-up) Well, `--accounts-db-caching-enabled` could be plumbed for `ledger-tool` as well like this `solana-validator`. For that I guess a bit of refactoring of `ProcessOptions` will be needed....

Reply:
hehe yeah I was hoping after accounts caching was stable, we would just remove the need for this option altogether, so I didn't bother plumbing it.